vmslave-config/pipeline/vmslave_config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+git diff --dirstat=files ${GIT_PREVIOUS_COMMIT} | grep "vmslave-config/"
+if [ $? -eq 0 ]; then
+    mv vmslave-config/pipeline/all vmslave-config/ansible/group_vars/all
+    cd vmslave-config/ansible
+    ansible-playbook pr_gate_vmslave_config.yml
```

Comment:
IMO they are all needed in different situation.
a variable file is popular in ansible playbook. It's easy to use and easy to understand.

But sometimes you may need to pass some env relevant vars to ansible playbook.
At that time  passing var throw ```ansible-playbook pr_gate_vmslave_config.yml --extra-vars="k=v" may be better.

In our ci situation, a var file is good enough.